x-pack/filebeat/input/gcppubsub/config.go

```diff
@@ -72,8 +72,8 @@ func defaultConfig() config {
 	c.ForwarderConfig = harvester.ForwarderConfig{
 		Type: "gcp-pubsub",
 	}
-	c.Subscription.NumGoroutines = 1
-	c.Subscription.MaxOutstandingMessages = 1000
+	c.Subscription.NumGoroutines = 2
+	c.Subscription.MaxOutstandingMessages = 1600
```

Comment:
Please add a comment for the reasoning behind these constants.

Reply:
Added comments